### PR TITLE
Add Home Assistant MQTT Sensor Example to operations.rst

### PR DESCRIPTION
**Pull request by pixeldoc2000**
Home Assistant MQTT Sensor Example added

### CODE REVIEW

**Comment by spacemanspiff2007** [comment]
Thank you for the PR.
I think it makes more sense to move this example to a different part of the docs which could be named "smart home integration".
Since not all smart meters report all values and the topics depend on the config I think one or two example entries how to build it are sufficient and less misleading.
I've another change upcoming in the next couple of weeks - do you want to do it or should I do it and take this PR as input?

File: docs/operations.rst [hunk 1]
```diff
@@ -443,3 +443,108 @@ The reported value will be the weighted mean value of the last 30s.
...
+         icon: mdi:transmission-tower
+         unit_of_measurement: kWh
+         state_class: total
+         device_class: energy
+         availability:
+           topic: "sml2mqtt/status"
+           payload_available: "ONLINE"
+           payload_not_available: "OFFLINE"
+         operations:
+           refresh action: 24:00
```
Comment (spacemanspiff2007) [inline docs/operations.rst:480]: What does this do?

Comment (pixeldoc2000) [reply docs/operations.rst:480]: Uh, good catch. That's an sml2mqtt config, not home-assistant. Should be removed.

**Comment by spacemanspiff2007** [comment]
Implemented in current version

**Pull request closed without merging**